CreeDictionary/API/search/core.py

```diff
@@ -70,37 +70,3 @@ def serialized_presentation_results(self):
 
     def __str__(self):
         return f"SearchRun<query={self.query}>"
-
-
-def to_internal_form(user_query: str) -> InternalForm:
-    """
-    Convert text to the internal form used by the database entries, tries, FSTs, etc.
-
-    In itwêwina, the Plains Cree dictionary, this means SRO circumflexes.
-    """
-    return InternalForm(to_sro_circumflex(user_query))
```

Comment:
This was part of a half (quarter? eighth?) finished refactor that would make the codebase language-independent. Alas, it never went anywhere.